web/src/app/projects/view/view.component.ts

```diff
@@ -40,9 +40,7 @@ export class ViewProjectComponent implements OnInit {
     }
 
     ngOnInit(): void {
-        this.projectSubscription = this.projectService
-            .findOneById(this.project.uid)
-            .subscribe((project: ProjectModel) => {
+        this.projectSubscription = this.projectService.findOneById(this.project.uid).subscribe((project: ProjectModel) => {
```

Comment:
sorry, I didn't meant the whole line, I just mean the return in the subscribe

```typescript
.subscribe((project: ProjectModel) => this.project = project)
```

Reply:
Done !!